dart/lib/src/protocol/debug_meta.dart

```diff
@@ -7,10 +7,14 @@ class DebugMeta {
 
   final List<DebugImage> _images;
 
-  /// An immutable list of dynamic libraries loaded into the process (see below).
-  List<DebugImage> get images => List.unmodifiable(_images);
-
-  DebugMeta({this.sdk, List<DebugImage> images}) : _images = images;
+  /// The list of debug images contains all dynamic libraries loaded
+  /// into the process and their memory addresses.
+  /// Instruction addresses in the Stack Trace are mapped into the list of debug
+  /// images in order to retrieve debug files for symbolication.
+  List<DebugImage> get images => List.from(_images);
+
+  DebugMeta({this.sdk, List<DebugImage> images})
+      : _images = images != null ? List.unmodifiable(images) : null;
```

Comment:
its the opposite, the private field should be `modifiable`
the public one should be `unmodifiable`
the `ctor` should copy to be sure that the private field is `modifiable`

Reply:
✅